.tekton/multiarch-tuning-operator-push.yaml

```
@@ -7,11 +7,11 @@ metadata:
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
     pipelinesascode.tekton.dev/on-cel-expression:
-      event == "push" && target_branch == "main" && (".tekton/multiarch-tuning-operator-pull-request.yaml".pathChanged() || "apis/***".pathChanged() || "controllers/***".pathChanged() || "pkg/***".pathChanged() || "test/***".pathChanged() || "konflux.Dockerfile".pathChanged() || "go.mod".pathChanged() || "main.go".pathChanged() || "go.sum".pathChanged() || "trigger-konflux-builds.txt".pathChanged() )
+      event == "push" && target_branch == "v0.9" && (".tekton/multiarch-tuning-operator-pull-request.yaml".pathChanged() || "apis/***".pathChanged() || "controllers/***".pathChanged() || "pkg/***".pathChanged() || "test/***".pathChanged() || "konflux.Dockerfile".pathChanged() || "go.mod".pathChanged() || "main.go".pathChanged() || "go.sum".pathChanged() || "trigger-konflux-builds.txt".pathChanged() )
   creationTimestamp: null
   labels:
-    appstudio.openshift.io/application: multiarch-tuning-operator
-    appstudio.openshift.io/component: multiarch-tuning-operator
+    appstudio.openshift.io/application: multiarch-tuning-operator-v0.9
```

```suggestion
    appstudio.openshift.io/application: multiarch-tuning-operator-v0-9
```

Shouldn't this be v0-9? I believe Tekton will complain about dots in the names.
Similar to https://github.com/openshift/multiarch-tuning-operator/blob/fbc-4.16/.tekton/fbc-v4-16-pull-request.yaml

---

I think you are right according to the description in the ProjectDevelopmentStreamTemplate